flight/Libraries/sanitycheck.c

```diff
@@ -121,6 +122,19 @@ int32_t configuration_check()
 			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_FAILSAFE:
 				// always ok
 				break;
+			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_LQG:
+			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_ATTITUDELQG:
```

Comment:
It would be nice for all of these to be LQGLeveling for consistency.  Also would be nice to catch it in stabilized banks -- e.g. if you have LQG/LQG/Acro or something (and factor this test out to a function).